src/version.rs

```diff
@@ -23,87 +56,14 @@ impl Display for ChangeKind {
 }
 
 /// Pack all information we know about a change made to a version of a crate.
-#[derive(RustcEncodable, Ord, PartialOrd, Eq, PartialEq, Debug)]
+#[derive(Serialize, Deserialize, Ord, PartialOrd, Eq, PartialEq, Debug)]
 pub struct CrateVersion {
     /// The crate name, i.e. `clap`.
     pub name: String,
     /// The kind of change.
+    #[serde(rename="yanked")]
     pub kind: ChangeKind,
```

Comment:
This code would be a lot simpler if this was just a `bool` -- is there any reason this needs to be a custom enum?

Reply:
I think it was for the purpose of forcing most explicit code when consuming this field. [It just reads nicer than pure boolean logic](https://github.com/onur/docs.rs/blob/e03fd5c2a12fcfefc5ecc500aace8821ec314da7/src/docbuilder/queue.rs#L21), and given the previous way of decoding the serialized object, there was no disadvantage.
Now when seeing the overhead imposed when using serde, I would certainly have second-guessed it, but since the work is already done, I would just keep it that way and break the downstream less.